amundsen_application/api/v0.py

```diff
@@ -4,18 +4,30 @@
 from flask import current_app as app
 from flask.blueprints import Blueprint
 
+from amundsen_application.api.metadata.v0 import USER_ENDPOINT
+from amundsen_application.api.utils.request_utils import request_wrapper
 from amundsen_application.models.user import load_user
 
+REQUEST_SESSION_TIMEOUT = 10
+
 LOGGER = logging.getLogger(__name__)
 
 blueprint = Blueprint('api', __name__, url_prefix='/api')
 
 
 @blueprint.route('/auth_user', methods=['GET'])
 def current_user() -> Response:
-    if (app.config['AUTH_USER_METHOD']):
-        user = app.config['AUTH_USER_METHOD'](app)
+    if app.config['AUTH_USER_METHOD']:
+        user_info = app.config['AUTH_USER_METHOD'](app)
```

Comment:
I'm concerned about changing the return type of a config method for the sake of a single feature, unless the feature can't be implemented at all without the change. It's not conducive to maintaining backwards compatibility, but I recognize that we can get away with it in this case due to the nature of how this method is currently used.

I am also concerned about the change in our internal system that this change would imposes on an unrelated feature. I believe we should discourage the introduction of changes that affect unrelated features, especially since this method could have been written without needing to impose that change.

Reply:
Changing this back to return a User object.